timeless/views.py

```diff
@@ -13,17 +14,52 @@ class CommentView(CrudView):
          model = Comment
          url_lookup = "comment_id"
 
-    @todo #123:30min Continue with the implementation of CrudAPIView.
-     Implement post, put and delete methods. We should return json
-     representation of object model in methods.
+    @todo #221:30min Remove references to database objects. we should not tie
+     our view layer to our database objects. We should create some form of
+     abstraction to handle the model calls, for example,
+     model.query.get(object_id) that would decorate the real implementation,
+     eliminating the coupling here.
     """
     model = None
     url_lookup = None
 
-    def get(self, *args, **kwargs):
-        object_id = kwargs.get(self.url_lookup)
-        if object_id:
-            result = self.model.query.get(object_id)
-            if result is None:
-                abort(404)
-        return "", HTTPStatus.OK
+    """
+    @todo #221:30min Continue with the implementation of CrudAPIView.
+     Implement get, post, put and delete methods. We should return json representation of object model in methods. Use 
+     FakeModel for a fake database object, and implement the desired calls on FakeQuery to get, create, save / update 
+     and delete returnin the result. Don't forget to implement the tests too, to test if CrudAPIView code is being 
+     called and returning the expected objects; please refer to #221 and #222 for documentation. After that remove the 
+     ignore annotation from tests on test_crud_api.py.
+    """
+
+    def get(self):
+        pass
+
+    def post(self):
+        pass
+
+    def put(self):
+        pass
+
+    def delete(self):
+        pass
+
+
+class FakeModel(DB.Model):
```

Comment:
@paulodamaso I think if you would like to use these fake classes, it should not inherit `DB.Model`. Since SQL-Alchemy will create a real table in DB.